instrumentation/executors/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/executors/ExecutorsInstrumentationModule.java

```diff
@@ -13,9 +13,13 @@
 import java.util.List;
 
 @AutoService(InstrumentationModule.class)
-public class ExecutorInstrumentationModule extends InstrumentationModule {
-  public ExecutorInstrumentationModule() {
-    super("executor");
+public class ExecutorsInstrumentationModule extends InstrumentationModule {
+
+  public ExecutorsInstrumentationModule() {
+    super(
+        "executors",
+        // TODO: remove that after release 1.26.0
+        "executor");
```

Comment:
What are the downsides to keeping both and leaving the old one as "undocumented"?